eos_data_distribution/ndn/http.py

```diff
@@ -57,14 +57,13 @@ def read_bytes_async():
     read_bytes_async()
 
 
-def fetch_http_headers(session, url):
+def fetch_http_headers(session, url, callback):
```

Comment:
we thought we were dropping packets because of time spent in soup, this commit was to test that theory, it needs to be repurposed.

we use it from http and manifest producers, it was a way to wrap those.